fixtures/rjs/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "systemjs-builder-test",
```

Comment:
I guess you meant to name this "rjs-test". Apart from that, there's not much I can contribute here I'm afraid. :-(